sw/airborne/math/pprz_algebra_int.h

```diff
@@ -608,6 +608,14 @@ static inline void int32_vect_zero(int32_t *a, const int n)
   for (i = 0; i < n; i++) { a[i] = 0.; }
 }
 
+/** a = v */
+static inline void int32_vect_value(int32_t *a, const int32_t v, const int n)
```

Comment:
Not very descriptive function name. Maybe something like set_value. Also the function description is not accurate (see vect_copy below). I think you mean something like a = v * I(n,1)...